src/components/table/LightTable.tsx

```diff
@@ -419,19 +401,13 @@ function LightTable({
           </Dialog>
         </Paper>
       )}
-      {loading && !rows?.length && (
-        <LoadingSpinner
-          delay={delay}
-          delayMessage="For large datasets, it can take a few minutes to fetch results. Thank you for your patience."
```

Comment:
Curious why you removed the delay messages? Did it make for clunky state transitions?

Reply:
The delay variable is populated by a BigQuery-specific function that polled the status of the BQ query. So, since we're no longer getting this data directly from BQ and we don't yet have a way to check in on the status of a TDR request, I removed this.